hdf5/src/hl/filters.rs

```diff
@@ -517,11 +517,15 @@ impl Filter {
             Ok(filters)
         })
     }
+
+    pub(crate) fn requires_chunking(&self) -> bool {
+        COMP_FILTER_IDS.contains(&self.id())
```

Comment:
Just wondering, that's what h5py does as well? I.e. just check a list of hardcoded well-known filter ids (because obviously there's other third party ones...)

I guess there's not much else you can do because there's no supplied filter metadata

Reply:
I am not sure if a user-supplied filter needs to have filtering? The behaviour if the filter is not known is to pass the error through, worst case the user gets a confusing error

Reply:
I think this is pretty much the whole list of what people actually use: https://github.com/silx-kit/hdf5plugin/tree/main/src

Reply:
The list is different from ours but is similar https://github.com/h5py/h5py/blob/49d7b8cde724a69a50d6cd4b218df7a6ba141633/h5py/_hl/dataset.py#L89

Reply:
Should we do the same (+blosc) then?

Although I'm not entirely sure all the non-compression filters they have listed _actually_ require chunking 🤔

Reply:
From https://support.hdfgroup.org/HDF5/doc/H5.user/Filters.html it seems all filters build on chunked data, so the only valid `required_chunking` is the one that returns `true`